pkg/types/urlsmap.go

```diff
@@ -71,23 +74,19 @@ func (c URLsMap) Len() int {
 }
 
 // parse parses the given string and returns a map listing the values specified for each key.
-func parse(s string) map[string][]string {
+func parse(s string) (map[string][]string, error) {
 	m := make(map[string][]string)
-	for s != "" {
-		key := s
-		if i := strings.IndexAny(key, ","); i >= 0 {
-			key, s = key[:i], key[i+1:]
-		} else {
-			s = ""
-		}
-		if key == "" {
-			continue
-		}
-		value := ""
-		if i := strings.Index(key, "="); i >= 0 {
-			key, value = key[:i], key[i+1:]
+	if s == "" {
+		return m, nil
+	}
+	pairs := strings.Split(s, ",")
+	for _, pair := range pairs {
```

Comment:
ok. i guess your solution is stricter than the previous. so parsing a=b,,c=d will return an error?